apps/nsq_stat/nsq_stat.go

```diff
@@ -27,6 +27,8 @@ var (
 	channel          = flag.String("channel", "", "NSQ channel")
 	statusEvery      = flag.Duration("status-every", -1, "(deprecated) duration of time between polling/printing output")
 	interval         = flag.Duration("interval", 2*time.Second, "duration of time between polling/printing output")
+	httpConnectTimeout = flag.Duration("http-client-connect-timeout", 2*time.Second, "timeout for HTTP connect")
+	httpRequestTimeout = flag.Duration("http-client-request-timeout", 2*time.Second, "timeout for HTTP request")
```

Comment:
Didn't think of this before, but now that we're setting a real request timeout, which encapsulates connection time, we should set the default for request timeout something greater than connect, say `5s`?

cc @jehiah